packages/carousel/src/ArrowButton/ArrowButton.styles.js

```diff
@@ -0,0 +1,54 @@
+import styled from 'styled-components'
+import { getPaletteColor, Box } from 'pcln-design-system'
+
+const defaultStyles = (props) => `
+  padding: 0;
+  background: ${getPaletteColor('background.lightest')(props)};
+  border-radius: 100%;
+  border: none;
+  box-shadow: ${props.theme.shadows.md};
```

Comment:
@wesleymartin85 should this use the new box-shadow theme values?

Reply:
Updated